### DISCUSSION TITLE
[Vote ended on 2022-10-11] Remove mellanox.onyx from Ansible 8

### DISCUSSION THREAD

**Discussion by mariolenz** [ansible-community/community-topics#144] [Votes done]
`mellanox.onyx` looks unmaintained (https://github.com/ansible-community/community-topics/issues/136).

Please vote with a +1 to remove it from Ansible 8 or -1 to keep it.

**Comment by mariolenz** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by thedoubl3j** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by gundalow** [comment]
+1 

Subject to the following since the repo is under github.com/ansible-collections:

* README.md updated to clearly state (at the top) that this is maintained
* A release pushed to Galaxy based off the last release with just the updated README.md (no other CI, etc fixes)
* Any open issues in the repo get a notice saying that the collection is unmaintained and will be removed 

This should likely be added to our procedures for removing collections that we have `write` to
